example/log-label-example/README.md

```diff
@@ -0,0 +1,97 @@
+# Custom Log Label Examples
+
+Examples illustrating how to add custom labels to logs generated by `userale`.
+
+## Adding Custom Labels to Logs
+
+### Method 1
+
+Consider the following HTML:
+
+```html
+
+<div>
+    <button>New Feature</button>
+</div>
+```
+
+The following code snippet will add a custom field and send a log whenever the new feature button is clicked:
+
+```js
+document.addEventListener('click', function (e) {
+    if (e.target.innerHTML === 'New Feature') {
+        window.userale.map(log => ({...log, logType: 'custom', customLabel: 'New Feature'}));
```

Comment:
window.userale.map is fine for now. Eventually, I think we'll want to encourage (or promote) more usage of our NPM deployment option (vs script tag). If including UserALE.js via NPM package, exports do not have to be called as a window object, e.g., window.userale.map --> userale.map. I think this is fine for now as it's consistent usage with our "top-level" example (which highlights script-tag use case).